doc/index.rst

```diff
@@ -1,9 +1,32 @@
-The Plutus Platform
+Plutus Tools Developer Guide
 ===============================
 
-The Plutus Platform is an application development platform for the Cardano blockchain.
+Plutus Tools are a collection of off-chain infrastructure resources. Plutus Tools are built 
+for developers who are authoring web-based applications that implement smart contracts on the 
+Cardano blockchain. 
+
+The Plutus Tools Team supports the underlying resources that developers need who are 
```

Comment:
Is it better to refer to the contents of the repo rather than the team?

Reply:
Yes, probably so. "The Plutus Tools repository supports the underlying resources..."